src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -730,6 +735,352 @@ private MappedRecommendationForModel generateRecommendationBasedOnModel(Timestam
         return mappedRecommendationForModel;
     }
 
+
+
+    private static int getNumPodsForNamespace(Map<Timestamp, IntervalResults> filteredResultsMap) {
+        LOGGER.info("Size of Filter Map: " +  filteredResultsMap.size());
```

Comment:
Should be `debug`, I think and add it in this format:
`LOGGER.debug("Size of Filter Map: {}",  filteredResultsMap.size());`

Reply:
Missed updating the log level to debug earlier. Updated now and corrected the format as well.